manifests/repositories.pp

```diff
@@ -20,8 +20,8 @@
         }
       } else {
         yumrepo { 'htcondor-stable':
-          descr    => "HTCondor Stable RPM Repository for Redhat Enterprise Linux ${major_release}",
-          baseurl  => "http://research.cs.wisc.edu/htcondor/yum/stable/rhel${major_release}",
+          descr    => "HTCondor Stable RPM Repository for Redhat Enterprise Linux ${facts['os']['release']['major']}",
+          baseurl  => 'http://research.cs.wisc.edu/htcondor/yum/stable/rhel$releasever',
```

Comment:
Hi @olifre,
I can confirm that this PR works for SL7 (e.g. `Scientific Linux release 7.3 (Nitrogen)`)


However, it does not work for CentOS 7:
```
cat /etc/redhat-release 
CentOS Linux release 7.3.1611 (Core)
```
Before any change my htcondor baseurl is
`baseurl=http://research.cs.wisc.edu/htcondor/yum/stable/rhel7`
and after changing this to 
`baseurl=http://research.cs.wisc.edu/htcondor/yum/stable/rhel$releasever`
I get the error
```
http://research.cs.wisc.edu/htcondor/yum/stable/rhel7.3/repodata/repomd.xml: [Errno 14] HTTP Error 404 - Not Found
Trying other mirror.
```

It seems like the SL team kept the behaviour from SL6 while the CentOS team "fixed" it (when comparing to `yum update --releasever=X.Y`).